singer-io/tap-activecampaign
# TDL-16383 Refactor code to class based

# Description of change
- Converted dictionary of each stream to a class
- Reshuffled sync logic methods from sync.py to `ActiveCampaign` class in the stream.py
- Verified that there is no difference in the catalog and state file of older dict based code and newer code.
- Passed the integration test case locally.

**Note:**
- Removed integration test case line from `config.yml` as no tests available in this PR. 
- So, First PR [#11 ](https://github.com/singer-io/tap-activecampaign/pull/11) needs to be merged which contains all test cases. 
# Manual QA steps
 - Clone old code and newer code locally.
 - Run discover mode and compare catalog.
 - Run sync mode and verify the output.
 
Note:
# Risks
 - 
 
# Rollback steps
 - revert this branch
